nhais/nhais-env-example.yaml

```diff
@@ -6,3 +6,11 @@ NHAIS_RETRY_DELAY: "0.1"
 
 #User-specific env variables
 NHAIS_OUTBOUND_QUEUE_HOST: "localhost:5672/outbound"
+
+#common tests
+INTEGRATION_TEST_ASID: '9XXXXXXXXXXXXX' #put your ASID from OpenTest credentials
+MHS_ADDRESS: http://localhost
+SCR_ADDRESS: http://localhost:9000/
```

Comment:
INTEGRATION_TEST_ASID, MHS_ADDRESS, and SCR_ADDRESS are for the MHS adaptor. Are they needed here?